src/main/java/net/pms/network/RequestV2.java

```diff
@@ -1170,11 +1170,11 @@ private StringBuilder browseSearchHandler(BrowseSearchRequest requestMessage) {
 					uf.setFakeParentId(containerID);
 				}
 
-				if (uf.isCompatible(mediaRenderer) && (uf.getPlayer() == null
-					|| uf.getPlayer().isPlayerCompatible(mediaRenderer))
-					 // do not check compatibility of the media for items in the FileTranscodeVirtualFolder because we need
-					 // all possible combination not only those supported by renderer because the renderer setting could be wrong.
-					|| files.get(0).getParent() instanceof FileTranscodeVirtualFolder) {
+				if (uf.isCompatible(mediaRenderer) && (uf.getPlayer() == null ||
+						uf.getPlayer().isPlayerCompatible(mediaRenderer)) ||
+						// do not check compatibility of the media for items in the FileTranscodeVirtualFolder because we need
+						// all possible combination not only those supported by renderer because the renderer setting could be wrong.
+						files.get(0).getParent() instanceof FileTranscodeVirtualFolder) {
```

Comment:
```suggestion
				if (
					uf.isCompatible(mediaRenderer) &&
					(uf.getPlayer() == null || uf.getPlayer().isPlayerCompatible(mediaRenderer)) ||
					// do not check compatibility of the media for items in the FileTranscodeVirtualFolder because we need
					// all possible combination not only those supported by renderer because the renderer setting could be wrong.
					files.get(0).getParent() instanceof FileTranscodeVirtualFolder
				) {
```